### PR TITLE
Add a test for spinning up OVN Kubernetes Cluster

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
The wni needs to create a Windows VM on the OVNKubernetes
cluster created as part of CI, adding a job to validate
the e2e tests

### CODE REVIEW

**Comment by ravisantoshgudimetla** [comment]
https://github.com/openshift/windows-machine-config-operator/pull/48 merged which ensures sufficient tags are added to Windows VM created.

/cc @wking 

https://github.com/openshift/windows-machine-config-operator/pull/48

I still need to fix the `generated-config` but I want to see if the rehearse job is working fine

/test pj-rehearse

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.2.yaml [hunk 1]
```diff
@@ -9,6 +9,9 @@ build_root:
     name: release
     namespace: openshift
     tag: golang-1.12
+tag_specification:
+  name: "4.2"
+  namespace: ocp
```
Comment (wking) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.2.yaml]: nit: In the master analog, you have `tag_specification` sorted alphabetically.  Here, you have it before `promotion`.  I dunno if there is a consensus around positioning, or any CI to enforce alphabetical ordering or anything, but seems like we should at least be consistent within this PR ;).

**Comment by wking** [comment]
[4.2 rehearsal seems sad][1]:

```
make: *** No rule to make target `test-e2e-tools'. Stop.
```

[Also][2]:

```
make: *** No rule to make target `verify-all'.  Stop.
```

Do you expect those to pass?

[1]: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5267/rehearse-5267-pull-ci-openshift-windows-machine-config-operator-release-4.2-test-e2e-tools/4
[2]: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5267/rehearse-5267-pull-ci-openshift-windows-machine-config-operator-release-4.2-verify/5

**Comment by ravisantoshgudimetla** [comment]
> [4.2 rehearsal seems sad](https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5267/rehearse-5267-pull-ci-openshift-windows-machine-config-operator-release-4.2-test-e2e-tools/4):
> 
> ```
> make: *** No rule to make target `test-e2e-tools'. Stop.
> ```
> 
> [Also](https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5267/rehearse-5267-pull-ci-openshift-windows-machine-config-operator-release-4.2-verify/5):
> 
> ```
> make: *** No rule to make target `verify-all'.  Stop.
> ```
> 
> Do you expect those to pass?

Nope, I don't expect it to pass.

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.2.yaml (hunk 1 above)
Comment (ravisantoshgudimetla) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.2.yaml]: Addressed. Thanks for catching but AFAIK, there is no consensus on positioning or enforcing alphabetical ordering, perhaps @stevekuznetsov knows it.

**Comment by ravisantoshgudimetla** [comment]
Removed unnecessary tests, @wking can you PTAL - I need your approval for the template changes :)

**Pull request merged**